### PR TITLE
Remove expr/stmt

### PR DESCRIPTION
**Pull request by krux02**
basically I tried to remove all occurences of ``stmt`` and ``expr`` from the entire repository. It is deprecated and when it is eventually removed this is a necessary step. I did not remove every single instance. I left the example from the talk as they were, because they are from a talk. But I tried to convert all the test cases to use the new symbols. I also tried to get rid of ``{.immediate.}`` macros/templates, but not everything was possible.

### CODE REVIEW

**Comment by krux02** [comment]
The tests are still failing, because I edited https://github.com/krux02/Nim/blob/remove-expr/tests/overload/tparams_after_varargs.nim so that it does fail. It has a problem that is also reported here: https://github.com/nim-lang/Nim/issues/5855

**Comment by Araq** [comment]
Ok, now what? Can't update the test to be green again?

**Comment by krux02** [comment]
Actually I can update the test to be green again. It is just that the test shows something that should work at some point in time. I would like to know how to mark that region of code, so that it doesn't become forgotten forever. For example a TODO comment?

**Comment by Araq** [comment]
> . For example a TODO comment?

Add a test that is ``disabled``? Or add an ``XXX`` comment.

**Comment by krux02** [comment]
Can this PR be merged? Is there interest to do so?

File: compiler/nim.cfg [hunk 1]
```diff
@@ -5,7 +5,6 @@ path:"llvm"
 path:"$projectPath/.."
 
 define:booting
-#import:"$projectpath/testability"
```
Comment (Araq) [inline compiler/nim.cfg]: @zah Is this removal fine with you?

**Comment by Araq** [comment]
Yes there is interest but long PRs require more review time than short PRs. ;-)

File: compiler/nim.cfg (hunk 1 above)
Comment (zah) [inline compiler/nim.cfg]: I don't use the testability module often, but I don't see why it should be removed.

Comment (krux02) [inline compiler/nim.cfg]: I just removed it in a habit to remove commented code. But if it causes discussion I reanimate it.

**Comment by krux02** [comment]
@Araq how far is the review process going?

**Pull request merged**